### PR TITLE
Delete Profile Title

### PR DESCRIPTION
**Pull request by esmaoui**

### CODE REVIEW

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/HomePageLocator.java [hunk 1]
```diff
@@ -684,4 +684,14 @@
...
+
+  public static final SelenideElement ELEMENT_USER_PROFILE                                    =
+          $(byXpath("//*[@id=\"UIUserPlatformToolBarPortlet\"]/ul/li[4]/a"));
+
+
+
+
+
+
+
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/HomePageLocator.java]: Useless blank lines

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/social/SocialLocator.java [hunk 2]
```diff
@@ -2356,4 +2356,12 @@
...
   public static final By ELEMENT_TASK_TAB=byClassName("uiIconAppTaskManagementApplication");
 
+
+  public static final By ELEMENT_EDIT_PROFILE= byClassName("uiIconEdit");
+  public static final By ELEMENT_TITLE_INPUT= byClassName("textarea");
+  public static final By ELEMENT_BUTTON_SAVE= byXpath("//*[@id=\"UIEditUserProfileForm\"]/div[3]/button[1]");
+  public static final By ELEMENT_ABOUT_ME= byClassName("simpleBox");
+
+
+
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/social/SocialLocator.java]: Useless blank lines

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/HomePagePlatform.java [hunk 3]
```diff
@@ -249,5 +250,16 @@ public void checkDisplayInSuggestionGadget(String user, boolean isPresent) {
...
+ */
 
+public void goToUserProfile(){
+
+  info("-- Go to user profile --");
+
+  $( ELEMENT_ACCOUNT_NAME_LINK).click();
+  $(ELEMENT_USER_PROFILE).click();
+
+}
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/HomePagePlatform.java]: Indentation is not good.

File: commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java [hunk 4]
```diff
@@ -82,10 +82,10 @@ public void afterEach() {
...
   }
 
-  @AfterEach
+ /* @AfterEach
   public void close() {
     if (!title().equals("Login")) {
       Selenide.close();
     }
-  }
+  }*/
```
Comment (thomasdelhomenie) [inline commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java]: Why this change ? Shouldn't we keep this to make sure the user is loggued in ? (cc @ilyeskallala )

Comment (ilyeskallala) [reply commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java]: This change to make sure that the user is loggued out. if it's not the case, the browser is closed and reopened for the next TC in login page.

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/HomePageLocator.java (hunk 1 above)
Comment (aboughdiri) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/HomePageLocator.java]: Done

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/social/SocialLocator.java (hunk 2 above)
Comment (aboughdiri) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/social/SocialLocator.java]: Fixed

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/HomePagePlatform.java (hunk 3 above)
Comment (aboughdiri) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/HomePagePlatform.java]: Fixed

File: commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java (hunk 4 above)
Comment (thomasdelhomenie) [reply commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java]: So we keep this change ? If so, no need to keep it in comments, we can delete it.
